_posts/2020-11-08-the-sunk-cost-fallacy.md

```diff
@@ -0,0 +1,110 @@
+
+## T + 2 - Fire
+
+I was looking at my coffee mug. The coffee looked extra dark today. My mind was boggling with one hypothesis and I was trying to convince myself that it's a lie. The coffee was not dark. My thoughts were.
+
+Z: “Hey! You okay?”
+
+My colleague was standing beside me without his laptop. He was seemingly not happy.
+
+Z: “I think this is a bug, it should not work like this.”
+
+Me: “It is correct, It's part of the feature.”
+
+Z: “We did not discuss this in the design meeting.”
+
+Me: “We should have discussed this, I remember it being documented.”
+
+I checked the documentation and this part of the feature was not to be found anywhere. My two worst fears had come true. 
+
+- I didn't document the meeting notes of my KT and this part of the feature somehow skipped being documented in design notes.
+
+- Every team member working on this feature has a different understanding of the feature.
+
+## T + 4 - Let it go
+
+I was walking by the river. The sky was looking beautiful with clouds of different shapes chatting with each other and the air passing through the meadow beneath seemed like it was whispering something to me. I closed my eyes focusing on its voice. 
+
+_**“Let it go”**_.
+
+I opened my eyes. The world was on fire. No, not on literal fire but an abstract concept of fire known to mankind as COVID. Knowledge transfer became difficult, most of the time people were on mute. Pair programming was no longer sitting beside your colleagues and watching them code but it was highly dependent on your house’ internet connectivity. 
+
+Progress of the feature seemed like a hallucination. In pursuit of scalability and adaptability, the design had become complicated, to begin with. The initial team was slowly replaced by new team members as a result of inflow of new complicated projects to work upon. We had already crossed the deadlines. There was no escape from this “black hole”.
+
+I fetched myself a cup of coffee and sat on my desk and opened my laptop. My dream was still lingering in my mind and I was not sure what it meant. I looked at the code. A thought came to me. 
+
+"Why not rewrite the implementation code of this feature with a much simpler and leaner design ?."
+
+But another thought hit me which threw me into a dilemma.
+
+“We invested so much time working on this for a couple of months. Even though the design is complicated, we should continue following it since we are already midway. It will anyway fulfil the requirement even if it takes longer. “
```

Comment:
```suggestion
“We have invested so much time working on this for a couple of months. Even though the design is complicated, we should continue following it since we are already midway. It will anyway fulfil the requirement even if it takes longer. “
```